_posts/2020/2020-10-20-introdution-to-vscode.md

```diff
@@ -0,0 +1,155 @@
+---
+layout: post
+title: VSCode shortcuts
+date: 2020-10-23
+header_image: public/shortcuts.jpg
+header_position: center
+header_overlay: true
+category: methods-and-tools
+tags: ["text", "editor", "vscode", "shortcuts"]
+authors: ["German"]
+about_authors: ["gsanemeterio"]
+---
+
+As a developer, I've used a lot of text editor such as Atom, Sublime Text, Notepad++... But it was with Visual Studio Code where I see a powerful tool for making websites.
+
+Frequently, I'm asked by my team, about a shortcut to do something. In this blog post, I'm going to share all of shortcuts that I use
+
+### Code Formating
+
+This shortcut helps to put your code in the correct format. It's necessary to set well your editor settings. To make it work, you have to press:
+
+* On Windows: `Shift + Alt + F`
+* On Mac: `⇧ + ⌥ + F`
+* On Linux: `Ctrl + Shift + I`
+
+{% image_custom image="/assets/img/pages/blog/images/vscode_shortcuts_format_code.gif" width="50" lightbox %}
+
+### Show Command Palette
+
+This is one of the most useful commands that I know. With this command, you can run an extension or even know his shortcuts.
```

Comment:
Suggestion: With this command, you can run an extension or even know the shortcuts of the extension.